src/lib/content/sodium/modules/Config.js

```diff
@@ -467,18 +467,23 @@ Config.ui.dtv = {
 
 // ABEMA
 Config.ui.abematv = {
-  target: `[class$="__player"], .com-vod-VODScreen-container`,
-  style: `#${Config.ui.id} {
-  position: absolute;
-  z-index: 1000001;
-  top: 12px;
-  left: 12px;
-  transition: 200ms;
-}
-[class$="__player"] > .com-tv-TVScreen__overlay--cursor-hidden ~ #${Config.ui.id},
-.com-vod-VODScreen-container--cursor-hidden > #${Config.ui.id} {
-  opacity: 0;
-}`,
+  target: `[class$=".com-tv-TVScreen__player-container"], .com-vod-VODScreen-container`,
```

Comment:
CSSのクラスセレクターはこのように書きます:

```suggestion
  target: `.com-tv-TVScreen__player-container, .com-vod-VODScreen-container`,
```

参考文献: [クラスセレクター \- CSS: カスケーディングスタイルシート \| MDN](https://developer.mozilla.org/ja/docs/Web/CSS/Class_selectors)